Boolean.CSharp.Test/CoreTests.cs

```diff
@@ -3,22 +3,126 @@
 
 namespace Boolean.CSharp.Test
 {
+    /*
+        TODO: GÖR KLART DOMAIN MODEL UTEFTER KLASSERNA
+        REVERSA BANKSTATEMENT
+        GÖR KLART KLASSDIAGRAM
+    */
     [TestFixture]
     public class CoreTests
     {
-        private Core _core;
+        private Bank _bank;
+        private SavingsAccount _savingsAccount;
+        private CurrentAccount _currentAccount;
+        private Customer _customer;
+        private BankStatementBuilder _bankStatementBuilder;
 
-        public CoreTests()
+        [SetUp]
+        public void Setup()
         {
-            _core = new Core();
+            _customer = new Customer();
+            _bankStatementBuilder = new BankStatementBuilder();
```

Comment:
i think the bankstatementbuilder should not be stored inside the objects bank and account, instead it should be created on the fly inside the method to create the bank statement and that's it